### PR TITLE
Vn-OKTA-396302-Add note about issuer to Node Express Quickstart Guide

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Added custom Authorization Server Id option to SDK sample snippet. Fixed 2 broken links in the Sign users in to web app > Configure packages  pages.
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-396302](https://oktainc.atlassian.net/browse/OKTA-396302)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnet/configmid.md [hunk 1]
```diff
@@ -37,7 +37,14 @@ public class Startup
     }
 }
 ```
-Okta middleware uses the default Custom Authorization Server by default. If you are using an Authorization Server other than the `default`, make sure to configure it by setting up the `AuthorizationServerId`. If you are using the [Org Authorization Server](/docs/concepts/auth-servers/), your middleware configuration should look like this: 
+
+Okta middleware uses the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server) by default. Ensure you configure `AuthorizationServerId` for the Authorization Server you are using for your app:
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnet/configmid.md:41]:
```suggestion
Okta middleware uses the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server) by default. Ensure you configure `AuthorizationServerId` for the Authorization Server that you are using for your app:
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnet/configmid.md [hunk 2]
```diff
@@ -37,7 +37,14 @@ public class Startup
...
+
+* if you are using the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server), set `AuthorizationServerId = "default"`
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnet/configmid.md:43]:
```suggestion
* If you are using the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server), set `AuthorizationServerId = "default"`.
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnet/configmid.md [hunk 3]
```diff
@@ -37,7 +37,14 @@ public class Startup
...
+* if you are using your own [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), set `AuthorizationServerId = "{authServerId}"`
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnet/configmid.md:44]:
```suggestion
* If you're using your own [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), set `AuthorizationServerId = "{authServerId}"`.
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnet/configmid.md [hunk 4]
```diff
@@ -37,7 +37,14 @@ public class Startup
...
+* if you are using the [Org Authorization Server](/docs/concepts/auth-servers/#org-authorization-server), set `AuthorizationServerId = ""`
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnet/configmid.md:45]:
```suggestion
* If you're using the [Org Authorization Server](/docs/concepts/auth-servers/#org-authorization-server), set `AuthorizationServerId = ""`.
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnet/configmid.md [hunk 5]
```diff
@@ -37,7 +37,14 @@ public class Startup
...
+
+Here is an example of middleware configuration using the Org Authorization Server:
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnet/configmid.md:47]:
```suggestion
Here's an example of the middleware configuration that uses the Org Authorization Server:
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnetcore3/configmid.md [hunk 6]
```diff
@@ -30,7 +30,13 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
-Okta middleware uses the default Custom Authorization Server by default. If you are using an Authorization Server other than the `default`, make sure to configure it by setting up the `AuthorizationServerId`. If you are using the [Org Authorization Server](/docs/concepts/auth-servers/), your middleware configuration should look like this: 
+Okta middleware uses the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server) by default. Ensure you configure `AuthorizationServerId` for the Authorization Server you are using for your app:
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnetcore3/configmid.md:33]:
```suggestion
Okta middleware uses the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server) by default. Ensure you configure `AuthorizationServerId` for the Authorization Server that you are using for your app:
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnetcore3/configmid.md [hunk 7]
```diff
@@ -30,7 +30,13 @@ public void ConfigureServices(IServiceCollection services)
...
+
+* if you are using the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server), set `AuthorizationServerId = "default"`
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnetcore3/configmid.md:35]:
```suggestion
* If you're using the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server), set `AuthorizationServerId = "default"`.
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnetcore3/configmid.md [hunk 8]
```diff
@@ -30,7 +30,13 @@ public void ConfigureServices(IServiceCollection services)
...
+* if you are using your own [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), set `AuthorizationServerId = "{authServerId}"`
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnetcore3/configmid.md:36]:
```suggestion
* If you're using your own [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), set `AuthorizationServerId = "{authServerId}"`.
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnetcore3/configmid.md [hunk 9]
```diff
@@ -30,7 +30,13 @@ public void ConfigureServices(IServiceCollection services)
...
+* if you are using the [Org Authorization Server](/docs/concepts/auth-servers/#org-authorization-server), set `AuthorizationServerId = ""`
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnetcore3/configmid.md:37]:
```suggestion
* If you're using the [Org Authorization Server](/docs/concepts/auth-servers/#org-authorization-server), set `AuthorizationServerId = ""`.
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnetcore3/configmid.md [hunk 10]
```diff
@@ -30,7 +30,13 @@ public void ConfigureServices(IServiceCollection services)
...
+
+Here is an example of middleware configuration using the Org Authorization Server:
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnetcore3/configmid.md:39]:
```suggestion
Here's an example of the middleware configuration that uses the Org Authorization Server:
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/go/configmid.md [hunk 11]
```diff
@@ -3,11 +3,13 @@ Update your environment variables to include the following:
 ```bash
 CLIENT_ID={yourClientId}
 CLIENT_SECRET={yourClientSecret}
-ISSUER=https://{yourOktaDomain}.com/oauth2/default
+ISSUER=https://{yourOktaDomain}.com/oauth2/{authServerId}
 ```
 
+If you are using the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server), set `{authServerId}=default`. If you are using another [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), set `{authServerId}` to the custom Authorization Server ID.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/go/configmid.md:9]:
```suggestion
If you're using the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server), set `{authServerId}=default`. If you're using another [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), set `{authServerId}` to the custom Authorization Server ID.
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/index.md [hunk 12]
```diff
@@ -13,6 +13,12 @@ You need the following values from the Okta Application and the Admin Console th
 * **Client Secret** &mdash; find it on the application's **General** tab.
 * **Okta Domain** &mdash; you can find the Okta Domain in the Admin Console's global header in the upper-right corner of the page. Click the section that displays your email and company name.  A drop-down box appears and displays general org information including the full Okta domain (for example, subdomain.okta.com).
 
+You can use the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server) that is provided in your [Okta Developer Edition](https://developer.okta.com/signup/) org. If you want to use another [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), then you need the following value from the Admin Console:
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/index.md:16]:
```suggestion
You can use the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server) that's provided in your [Okta Developer Edition](https://developer.okta.com/signup/) org. If you want to use another [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), then you need the following value from the Admin Console:
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/nodeexpress/configmid.md [hunk 13]
```diff
@@ -22,3 +22,5 @@ const oidc = new ExpressOIDC({
 // ExpressOIDC attaches handlers for the /login and /authorization-code/callback routes
 app.use(oidc.router);
 ```
+
+If you are using the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server), set `{authServerId}=default`. If you are using another [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), set `{authServerId}` to the custom Authorization Server ID.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/nodeexpress/configmid.md:26]:
```suggestion
If you're using the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server), set `{authServerId}=default`. If you're using another [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), set `{authServerId}` to the custom Authorization Server ID.
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/php/configmid.md [hunk 14]
```diff
@@ -3,6 +3,7 @@ Update your environment variables to include the following:
 ```bash
 CLIENT_ID={yourClientId}
 CLIENT_SECRET={yourClientSecret}
-ISSUER=https://{yourOktaDomain}.com/oauth2/default
+ISSUER=https://{yourOktaDomain}.com/oauth2/{authServerId}
 ```
 
+If you are using the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server), set `{authServerId}=default`. If you are using another [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), set `{authServerId}` to the custom Authorization Server ID.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/php/configmid.md:9]:
```suggestion
If you're using the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server), set `{authServerId}=default`. If you're using another [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), set `{authServerId}` to the custom Authorization Server ID.
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/springboot/configmid.md [hunk 15]
```diff
@@ -1,12 +1,14 @@
...
+okta.oauth2.issuer=https://${yourOktaDomain}/oauth2/{authServerId}
 okta.oauth2.client-id={clientId}
 okta.oauth2.client-secret={clientSecret}
 
 # Customize the callback route path
 okta.oauth2.redirect-uri=/authorization-code/callback
 ```
 
-Take a look at the [Spring Boot Externalized Configuration](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html) for details on other ways to configure properties.
+If you are using the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server), set `{authServerId}=default`. If you are using another [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), set `{authServerId}` to the custom Authorization Server ID.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/springboot/configmid.md:12]:
```suggestion
If you're using the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server), set `{authServerId}=default`. If you're using another [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), set `{authServerId}` to the custom Authorization Server ID.
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/springboot/configmid.md [hunk 16]
```diff
@@ -1,12 +1,14 @@
...
+
+Take a look at the [Spring Boot Externalized Configuration](https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config) for details on other ways to configure properties.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/springboot/configmid.md:14]:
```suggestion
Take a look at the [Spring Boot Externalized Configuration](https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config) for details on other ways to configure the properties.
```

**Review by christinechoy-okta — COMMENTED** [review]
Just a few comments...thanks!

**Review by bretterer — APPROVED** [review]

**Review by aarongranick-okta — APPROVED** [review]

**Review by christinechoy-okta — APPROVED** [review]

**Pull request merged**